site/_data/lib/find.js

```diff
@@ -1,51 +1,42 @@
 const path = require('path');
 
-const cacheKey = Symbol('find-cache');
-const alreadyNotifiedKey = Symbol('already-notified');
+const urlCacheKey = Symbol('find-cache-url');
+const pathCacheKey = Symbol('find-cache-path');
+
+const defaultLocale = 'en';
 
 /**
- * Builds and uses a cache to ensure fast lookup of Eleventy pages.
+ * Builds and uses a cache to ensure fast lookup of Eleventy pages by
+ * a specified key.
  *
  * @param {EleventyCollectionItem[]} collection
- * @param {string} urlToFind
+ * @param {unique symbol} cacheKey
+ * @param {string} key
+ * @param {string} needle
  * @return {EleventyCollectionItem|undefined}
  */
-const internalFind = (collection, urlToFind) => {
-  /** @type {{[url: string]: EleventyCollectionItem}} */
+const internalFind = (collection, cacheKey, key, needle) => {
```

Comment:
Please add tests in https://github.com/GoogleChrome/developer.chrome.com/blob/main/tests/site/_data/lib/find.js

Reply:
Added tests for the new `findByFilePath` methods. That should cover all uses of `internalFind`.